tests/scale_up_status_updates/scale_up_status_updates_suite_test.go

```diff
@@ -0,0 +1,60 @@
+package scale_up_status_updates
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	ginkgo_util "github.com/datastax/cass-operator/mage/ginkgo"
+	"github.com/datastax/cass-operator/mage/kubectl"
+)
+
+var (
+	testName     = "Scale up status updates"
+	namespace    = "test-scale-up-status-updates"
+	dcName       = "dc1"
+	dcYaml       = "../testdata/oss-two-rack-six-node-dc.yaml"
+	operatorYaml = "../testdata/operator.yaml"
+	dcResource   = fmt.Sprintf("CassandraDatacenter/%s", dcName)
+	dcLabel      = fmt.Sprintf("cassandra.datastax.com/datacenter=%s", dcName)
+	ns           = ginkgo_util.NewWrapper(testName, namespace)
+)
+
+func TestLifecycle(t *testing.T) {
+	AfterSuite(func() {
+		logPath := fmt.Sprintf("%s/aftersuite", ns.LogDir)
+		kubectl.DumpAllLogs(logPath).ExecV()
+		fmt.Printf("\n\tPost-run logs dumped at: %s\n\n", logPath)
+		ns.Terminate()
+	})
+
+	RegisterFailHandler(Fail)
+	RunSpecs(t, testName)
+}
+
+var _ = Describe(testName, func() {
+	Context("when in a new cluster", func() {
+		Specify("the operator can scale up a datacenter", func() {
```

Comment:
Might want to make this a bit more descriptive for what you are actually testing. We have a couple of other scale up tests that do nothing but increase the node count, and we probably want to distinguish the messages in this one.

Reply:
yeah, I just copied/pasted from the scale up test. I will update this.

Reply:
@sandoichi wdyt about the updated message?

Reply:
Looks good